cmake/doxygen.cmake

```diff
@@ -35,17 +31,20 @@ macro(build_doxygen_documentation)
     # Create the doxyfile in function of the current project.
     # If the Doxyfile.in does not exists, the cmake step stops.
     configure_file(${MPI_CMAKE_MODULES_RESOURCES_DIR}/Doxyfile.in
-                   ${doc_build_folder}/Doxyfile
-                   @ONLY IMMEDIATE)
+                    ${doc_build_folder}/Doxyfile
+                    @ONLY IMMEDIATE)
 
     # the doxygen target is generated
-    add_custom_target (${PROJECT_NAME}_doc ALL
-      COMMAND ${DOXYGEN_EXECUTABLE} ${doc_build_folder}/Doxyfile
-      SOURCES ${doc_build_folder}/Doxyfile
-      WORKING_DIRECTORY ${doc_build_folder})
+    add_custom_target (${PROJECT_NAME}_doxygen_html
+        COMMAND ${DOXYGEN_EXECUTABLE} ${doc_build_folder}/Doxyfile
+        SOURCES ${doc_build_folder}/Doxyfile
+        WORKING_DIRECTORY ${doc_build_folder})
 
     # install the documentation
-    install(DIRECTORY ${doc_build_folder} DESTINATION ${doc_install_folder})
-  endif(BUILD_DOCUMENTATION)
+    # install(DIRECTORY ${doc_build_folder} DESTINATION ${doc_install_folder})
```

Comment:
Is this intentionally commented?

Reply:
yes it is, the installation had a bit of trouble